pennylane/devices/default_qubit_jax.py

```diff
@@ -357,3 +357,43 @@ def _count_binned_samples(indices, batch_size, dim, bin_size, num_bins):
                     prob = prob.at[i, state, b].set(count / bin_size)
         # resize prob which discards the 'filled values'
         return prob[:, :-1]
+
+
+if has_jax:
```

Comment:
Is this required for the type hints? :thinking:

Reply:
also note that in this file there is no `has_jax` variable (currently causing errors)

Reply:
yes. If I let the type hints go I could avoid this...
Should I remove the hints?

Reply:
@albi3ro do we have a policy for this?

Reply:
Problematic type hints can be wrapped in strings:

```
        H_jax: "ParametrizedHamiltonianPytree", operation_data: Any
     ) -> Callable[["jax.Array", float, Tuple["jax.Array", ...]], "jax.Array"]:
```

This way we still have the information provided by the type hint, but it won't cause import problems

Reply:
Ok, done.